db/migrate/20211122155214_add_columns_to_users.rb

```diff
@@ -0,0 +1,13 @@
+class AddColumnsToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :first_name, :string
+    add_column :users, :last_name, :string
+    add_column :users, :address, :string
+    add_column :users, :language, :string
+    add_column :users, :description, :string
+    add_column :users, :link, :string
```

Comment:
As discussed, need to add more specific link:
add_column :users, :link_github, :string
add_column :users, :link_malt, :string
add_column :users, :link_linkedin, :string
add_column :users, :link_slack, :string